dimod/reference/samplers/exact_solver.py

```diff
@@ -159,6 +159,45 @@ def sample_poly(self, polynomial, **kwargs):
         """
         return ExactSolver().sample(polynomial, **kwargs)
 
+class ExactSolverDQM(Sampler):
+    """A simple exact solver for testing and debugging code using your local CPU.
+
+    Notes:
+        This solver becomes slow for problems with 18 or more
```

Comment:
Did we test at this size or is it just copied from the above?

Reply:
If we actually tested it, then a critical piece is missing -- number of cases per variable.

Reply:
The largest I tested was 10 variables and 4 cases, haven't tested it further. I assumed if BQM isn't feasible past 18, this can't handle beyond the product of number of cases over all variables is approx 2^18. They both calculate the energy of all possible configurations

Reply:
Right - I imagine this number should be smaller.